.cirrus.yml

```diff
@@ -1,12 +1,12 @@
 freebsd_instance:
-  image_family: freebsd-11-3-snap
+  image_family: freebsd-11-4-release
```

Comment:
You either should use `image` for `freebsd-11-4-release` or:

```suggestion
  image_family: freebsd-11-4
```

Reply:
Thanks for the tip, @fkorotkov .